### ISSUE TITLE
WWSympa crashes by topics.conf with inappropriate format

### ISSUE THREAD

**Issue by matrixbx** [sympa-community/sympa#1242]
Version 6.2.60
-------

Installation method deb
-------------------

Expected behavior
-----------------

just upgraded from debian buster to bullseye and got this error, cannot figure out what to do with it

---------------

Erreurs : bug in logic. Ask developer at /usr/share/sympa/lib/Sympa/Scenario.pm line 149.
Pile d'appel

DIED: bug in logic. Ask developer at /usr/share/sympa/lib/Sympa/Scenario.pm line 149.
 at /usr/share/sympa/lib/Sympa/Scenario.pm line 149.
	Sympa::Scenario::new('Sympa::Scenario', 'listes.aquilenet.fr', 'topics_visibility', 'name', undef) called at /usr/lib/cgi-bin/sympa/wwsympa.fcgi line 15737
	main::export_topics('listes.aquilenet.fr') called at /usr/lib/cgi-bin/sympa/wwsympa.fcgi line 2753
	main::check_param_in() called at /usr/lib/cgi-bin/sympa/wwsympa.fcgi line 1516

Additional information
----------------------

**Comment by ikedas** [comment]
Hi @matrixbx , could you please show the content of `topics.conf`?

**Comment by matrixbx** [comment]
Hi, here it is

Aquilenet/public
title Public

Aquilenet/private
title Private

Girondix/public
title Girondix public

Girondix/private
title Girondix private

Ressourceries/public
title Public

Ressourceries/private
title Private

**Comment by ikedas** [comment]
@matrixbx , when I corrected `topics.conf` as below, it works:
``` code
Aquilenet
title Aquilenet

Aquilenet/public
title Public

Aquilenet/private
title Private

Girondix
title Girondix

Girondix/public
title Girondix public

Girondix/private
title Girondix private

Ressourceries
title Ressourceries

Ressourceries/public
title Public

Ressourceries/private
title Private
```

Though the fact that wwsympa crashes due to incorrect input is a bug, but please give us some time to fix it.

**Comment by matrixbx** [comment]
Hi ikedas, thank you for the correction, it work for us too :)

**Comment by ikedas** [comment]
Fix will be included in the next release (perhaps 6.2.71b).  @matrixbx , thank you for reporting bug!
